pkg/virt-launcher/notify-client/client.go

```diff
@@ -238,9 +238,37 @@ func newWatchEventError(err error) watch.Event {
 	return watch.Event{Type: watch.Error, Object: &metav1.Status{Status: metav1.StatusFailure, Message: err.Error()}}
 }
 
+func humanReadableLibvirtEvent(event int) string {
```

Comment:
@davidvossel can you drop https://github.com/kubevirt/kubevirt/pull/9290/commits/5cd094ef2df93658d48e7f9f23b7f42238db3722?